### DISCUSSION TITLE
is clearlinux Unsuitable for use on production machines?

### DISCUSSION THREAD

**Discussion by gitcnd** [clearlinux/distribution#2887] [General]
Not being critical or rude - just observing (and posting here in case it helps others like myself save some time in future) - there's 887 issues right now, and so-far this year, only 47 closed (one per 2.5 days).

I notice a huge number of unanswered problems and questions in the forums https://community.clearlinux.org/ and I notice that google doesn't have any search results for most of those things either.

I've got an intel-branded server with intel motherboard and xeon CPUs, and it either boots to a black screen (like many others in forums), or if I fuss with blacklists I get half way into boot then it locks up with all the text on the screen flashing (like many others on forums)... so my observation here is that this distro is probably unsuitable for prime-time use because:

*. If this was a production box - there wouldn't be any way to easily find out how to fix anything, or fixes wouldn't arrive ?
*. Since this Intel product doesn't even boot on an intel branded machine, and after effort, does "weird things" instead of producing error messages, it seems likely that it's not being adequately tested, and unsuitable patches with questionable decisions are being included (e.g. make the whole screen flash, instead of print out  an error message).

Again - not trying to be mean or anything - just warning newcomers that this distro appears suitable only for hardcore dev types of user.

**Comment by fenrus75** [comment]
we work on many of the issues but generally we depend on the reporter to
close the issue as confirmed...
so likely most of the open issues that arent' feature requests are resolved
.. just not confirmed

And yes we're mostly server focused, which means especially nvidia client
graphics hardware is a struggle to get working well..
(and no we don't patch anything there)
